content/changelog/2024-05-28-tasks-in-console.md

```diff
@@ -15,7 +15,7 @@ description: Just a box to check
 excludeSearch: true
 ---
 
-Since [its release 3.5.2](/changelog/2024-04-11-clever-tools-3.5.2/), Clever Tools is able to manage [Clever Tasks](/doc/develop/tasks/). It's now possible to create, configure and deploy them directly from the Console. To declare an application as a Task, check the corresponding box in the `Information` tab.
+Since [its release 3.5.2](/changelog/2024-04-11-clever-tools-3.5.2/), Clever Tools is able to manage Clever Tasks. It's now possible to create, configure and deploy them directly from the Console. To declare an application as a Task, check the corresponding box in the `Information` tab.
```

Comment:
```suggestion

Since [its release 3.5.2](/changelog/2024-04-11-clever-tools-3.5.2/), Clever Tools is able to manage [Clever Tasks](/doc/develop/tasks/). It's now possible to create, configure and deploy them directly from the Console. To declare an application as a Task, check the corresponding box in the `Information` tab.
```